x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -849,24 +849,8 @@ process where modulo(11, add(serial_event_id, 1)) == serial_event_id'''
 expected_event_ids  = [1, 2, 3, 5, 11]
 description = "test built-in math functions"
 
-[[queries]]
```

Comment:
Can you add more tests here, please?
The way I see it, there are tests (just one) for no argument `number` function and for base 16. How about base 2, explicit base 10, and custom bases?

Reply:
Added tests for the processor, and added integration tests here
lob/867fc01a7fc39285c118792283e3457e4e1c85b3/x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml#L22-L39